storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -2357,3 +2357,41 @@ void check_dump_innodb_buffer_pool(MYSQL *connection) {
     xb_mysql_query(mysql_connection, change_bp_dump_pct_query, false);
   }
 }
+
+/* print tables that have INSTANT ADD/DROP column row version
+ * @param[in]   connection  MySQL connection handler
+ * @return true if tables with row versions > 0 */
+bool print_instant_versioned_tables(MYSQL *connection) {
+  /* PS not affected by INSTANT issues. Upstream only affected on 8.0.29+ */
+  if (server_flavor == FLAVOR_PERCONA_SERVER || mysql_server_version < 80029)
```

Comment:
We need to relax this in case upstream fixes it in 8.0.30. So can you please ensure we revisit by adding debug assert or a test failure?

Reply:
@rahulmalik87 added it as a test case. If oracle releases 8.0.30 with this fixed we still want to make sure that users wait for PXB 8.0.30 before starting to taking backups.
So I added it as a test error only to be caught as part of the merge.